alfresco-googledrive-repo-base/pom.xml

```diff
@@ -23,6 +23,12 @@
             <artifactId>alfresco-data-model</artifactId>
             <version>${dependency.alfresco-data-model.version}</version>
             <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>ch.qos.reload4j</groupId>
+                    <artifactId>reload4j</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Is it possible to bump the `alfresco-data-model` version to a version that does not package reload4j? I believe that ideally `alfresco-data-model` and the repository version should match since they're tagged together.

Reply:
done 👍